src/dotnet/Chat.Contracts/Backend/IChatAuthorsBackend.cs

```diff
@@ -23,9 +23,18 @@ public interface IChatAuthorsBackend : IComputeService
     [CommandHandler]
     Task<ChatAuthor> Create(CreateCommand command, CancellationToken cancellationToken);
 
+    [CommandHandler]
+    Task<ChatAuthor> ChangeHasLeft(ChangeHasLeftCommand command, CancellationToken cancellationToken);
+
     [DataContract]
     public sealed record CreateCommand(
         [property: DataMember] string ChatId,
         [property: DataMember] string UserId
         ) : ICommand<ChatAuthor>, IBackendCommand;
+
+    [DataContract]
+    public sealed record ChangeHasLeftCommand(
```

Comment:
Ideally there should be just one UpdateCommand, which applies all proposed changes. Proposed change = field value of command != null, which describes it.

The reasoning is simple: most of these actions will share nearly the same logic for pulling entity, running checks, etc.; same for the front-end. So why bother to create N commands if you can have just one?

Besides that, for any complex change you can create fields like `MoveAction? Move { get; init; }` inside `UpdateCommand`.

Reply:
Long story short, CRUD, where U can be responsible for all possible updates.

Reply:
Would you propose to define command like below and adapt handler?
```
[DataContract]
public sealed record UpdateCommand(
     [property: DataMember] string AuthorId,
     [property: DataMember] bool? HasLeft
) : ICommand<ChatAuthor>, IBackendCommand;
```